repo_checker.py

```diff
@@ -476,6 +481,8 @@ def get_optparser(self):
         parser = ReviewBot.CommandLineInterface.get_optparser(self)
 
         parser.add_option('--skip-cycle', action='store_true', help='skip cycle check')
+        parser.add_option('--force', action='store_true', help='force review even if project is not ready')
+        parser.add_option('--limit-group', metavar='GROUP', help='only review requests in specific group')
```

Comment:
Is this relevant for `ReviewBot` as could want to run any bot against requests in a group to get a staging finished? Although currently this is likely the only bot to be waiting on besides legal.

Reply:
could be refactored into reviewbot indeed